splinepy/microstructure/tiles/ellips_v_oid.py

```
@@ -27,6 +30,18 @@ class EllipsVoid(_TileBase):
     _para_dim = 3
     _evaluation_points = _np.array([[0.5, 0.5, 0.5]])
     _n_info_per_eval_point = 4
+    _sensitivities_implemented = True
+    # TODO: clever parameter bounds and checks if given parametrization would
```

Pylint (3.3.7)

[warning] TODO: clever parameter bounds and checks if given parametrization would

(W0511)